image_classification/index.html

```diff
@@ -150,6 +150,10 @@
       </div>
     </div>
   </div>
+  <script>
+    // Set module as undefined to ensure loading all 3rd-party libraries under browser environment even when we reuse codes for electron.
+    if (typeof module === "object") module = undefined;
+  </script>
```

Comment:
I guess this workaround is to fix jquery loading issue in electron, is it? If it is, please mention it in your comment. And you probably need to restore the module after the jquery loads.

Refer to: https://stackoverflow.com/questions/32621988/electron-jquery-is-not-defined

Reply:
Thanks! I have updated in this [commit](https://github.com/webmachinelearning/webnn-samples/pull/49/commits/45226dda75de1f8e15d4645e1f5044ea12d4cb85). PTAL. @huningxin